ui/app/components/keyboard-shortcuts-modal.hbs

```diff
@@ -30,7 +31,7 @@
                 {{/if}}
               {{/if}}
 
-              <button disabled={{or (not command.rebindable) command.recording}} type="button" {{on "click" (action this.keyboard.rebindCommand command)}}>
+              <button class="re-bind" disabled={{or (not command.rebindable) command.recording}} type="button" {{on "click" (action this.keyboard.rebindCommand command)}}>
```

Comment:
`nit`: This class is misleading. The class is being added to this button to be used for a test assertion. Personally, I prefer using a `data-test` attribute for this. Even if you're inside of an `#each` you can set the value of the attribute so you can query the correct DOM node. But, if that doesn't work, you can apply an element id which is a better querying pattern.

Disregard this comment if this is actually an applied style, but I checked the `12762-ui-general-keyboard-navigation` and didn't see it there.

Reply:
Got it; modified to a `data-test-rebinder` attribute.